llvm/test/Analysis/UniformityAnalysis/AMDGPU/always_uniform.ll

```diff
@@ -58,6 +67,18 @@ define void @single_lane_func_arguments(i32 %i32, i1 %i1) #2 {
  ret void
 }
 
+; CHECK-LABEL: for function 'divergent_args':
+; CHECK: DIVERGENT ARGUMENTS
+define void @divergent_args(i32 %i32, i1 %i1) {
+ ret void
+}
+
+; CHECK-LABEL: for function 'no_divergent_args_if_inreg':
+; CHECK-NOT: DIVERGENT ARGUMENTS
```

Comment:
don't need ARGUMENTS?

Reply:
Because this is a "CHECK-NOT", removing the word "ARGUMENTS" would make the condition stronger. So I suppose it can be removed.